packages/bitcore-client/bin/wallet-derive-address

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env node
+
+'use strict';
+
+const program = require('../ts_build/program');
+const { Wallet } = require('../ts_build/wallet');
+const promptly = require('promptly');
+
+program
+  .version(require('../package.json').version)
+  .option('--name <name>', 'REQUIRED - Wallet Name')
+  .option('--change [change]', 'optional - should derive change addresses')
+  .option('--addressIndexes [addressIndexes]', 'optional - takes one or more indexes')
```

Comment:
I believe the request was for a range (i.e. 4-20) vs an array (i.e. [4, 34, 40, 6]) I'm not sure the use case for deriving from an array of indexes but it's just a thought.